Makefile

```diff
@@ -0,0 +1,13 @@
+clean:
+	rm -f *.rpm
+
+rpm:
+	mkdir -p packaging/root/usr/bin
+	cp *.py packaging/root/usr/bin
+	fpm -s dir -t rpm \
+            -n python-snappy \
```

Comment:
There is already a package called `snappy`, so we can't use that, and by convention, `python-snappy` would imply a Python module, which this isn't.  Consider a different name such as `mozilla-snappy` or `snappy-symserv` or something like that.